result_pipe.py

```diff
@@ -0,0 +1,106 @@
+from livekit import rtc 
+from typing import Literal, Optional, Dict, Any, Callable
+from dataclasses import dataclass, field
+import uuid
+from datetime import datetime
+import json
+import time
+
+
+
+
+CLASSIFIER_RESULT_TOPIC = "classifier-result-topic"
+DETECTOR_RESULT_TOPIC = "detector-result-topic"
+TRACKER_RESULT_TOPIC = "tracker-result-topic"
+TOPICS_LIST = [CLASSIFIER_RESULT_TOPIC, DETECTOR_RESULT_TOPIC, TRACKER_RESULT_TOPIC]
+TOPICS_TYPE = Literal[CLASSIFIER_RESULT_TOPIC, DETECTOR_RESULT_TOPIC, TRACKER_RESULT_TOPIC]
```

Comment:
any particular reason these are denoted as "topics"? I'm assuming Livekit labels them this way?

Reply:
now seeing `publish_data()` expects a topic param

Reply:
yep livekit nomenclature